odoo/addons/openupgrade_records/models/comparison_config.py

```diff
@@ -47,11 +47,9 @@ def test_connection(self):
             ids = user_model.search([("login", "=", "admin")])
             user_info = user_model.read(ids[0], ["name"])
         except Exception, e:
-            raise UserError(
-                _("Connection failed."), unicode(e))
+            raise UserError(u'%s%s' % (_("Connection failed.\n\nDETAIL: "), str(e)))
```

Comment:
If you mean that translators are then confronted with %s codes in their strings, they should be used to that by now ;-)